servant-client/test/Servant/StreamSpec.hs

```diff
@@ -46,19 +48,23 @@ import           System.Entropy
                  (getEntropy, getHardwareEntropy)
 import           System.IO.Unsafe
                  (unsafePerformIO)
+{-
```

Comment:
It looks like the tests that used this import have been commented out for 4 years ; maybe we can just remove them ?